spec/lib/vcr/cassette/serializers_spec.rb

```diff
@@ -50,7 +50,7 @@ class Cassette
       it_behaves_like "a serializer", :yaml,  "yml",  :lazily_loaded do
         it_behaves_like "encoding error handling", :yaml, ArgumentError do
           let(:string) { "\xFA".force_encoding("UTF-8") }
-          before { ::YAML::ENGINE.yamler = 'psych' }
+          before { ::YAML::ENGINE.yamler = 'psych' if defined?(::YAML::ENGINE) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.